tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -1860,8 +1863,8 @@ namespace tuplex {
                 else hashmap_free(task_sink.hm); // remove hashmap (keys and buckets already handled)
 
                 // delete task
-                delete tasks[i];
-                tasks[i] = nullptr;
+//                delete tasks[i];
```

Comment:
why is this commented? bug?

Reply:
This is commented because it was leading to a bug. This delete was added during the open sourcing, as it was not on my old branch in the old repo. The bug was the following: executeTransformStage calls createFinalHashmap. createFinalHashmap ends up deleting the tasks. Later in executeTransformStage, if the webUI is enabled, we have to iterate through the tasks to get the number of output rows. However, this was causing a nullptr dereference. So I commented the delete tasks[I] and setting it to a nullptr. If the memory is not being managed elsewhere, the tasks can be deleted after the history server is done using them and then deleted in an else statement if webUI not enabled.